jena-fuseki2/jena-fuseki-ui/tests/unit/services/fuseki.service.spec.js

```diff
@@ -26,7 +26,9 @@ describe('FusekiService', () => {
   let fusekiService
   let clock
   beforeEach(async () => {
-    fusekiService = new FusekiService()
+    fusekiService = new FusekiService({
+      pathname: ''
+    })
```

Comment:
Instead of accessing `window.location` directly, did some quick inversion of control, to make it easier to test too... later we can have better tests with and without a pathname, by just passing a JS object that we have full control how it behaves (it is hard/annoying to mock global objects like `window.location`)